db/schema.rb

```diff
@@ -281,10 +281,10 @@
     t.string "last_sign_in_ip"
     t.integer "role"
     t.string "old_user_id"
+    t.string "phone"
```

Comment:
This looks like your DB has gotten out of sync. If you reset your database (`rake db:drop; rake db:create; rake db:migrate`), I think this goes away?

Reply:
aah maybe so I will give it a try